.monorepolint.config.mjs

```diff
@@ -88,21 +88,29 @@ export default {
     packageEntry({
       options: {
         entries: {
+          type: "module",
+          main: "dist/cjs/index.cjs",
+          module: "dist/esm/index.mjs",
+          types: "dist/esm/index.d.ts",
+          sideEffects: false,
+          publishConfig: {
+            access: "public",
+          },
           // @turf/turf is commonly consumed through CDNs, moving this output file is a breaking change for anyone
           // who has a hardcoded reference to this specific file, instead of letting the CDN pick the path.
           // Example of a URL that will break: https://unpkg.com/@turf/turf/dist/turf.min.js
           // Example of a URL that will keep working: https://unpkg.com/@turf/turf
           browser: "turf.min.js",
-          files: ["dist", "index.d.ts", "turf.min.js"],
+          files: ["dist", "turf.min.js"],
```

Comment:
module in this package specifically might interact weirdly with turf.min.js here because it should be a commonjs output.

Reply:
But won't that be treated as ESM now because the package is set to `module`?

Reply:
If we're talking about turf.min.js only, rollup will output UMD (CJS+AMD hybrid) regardless of what module is set to.

If you're talking about the overall @turf package, the exports section in package.json should allow node to pick the appropriate dist/ flavour for whatever environment the caller is trying to import into.

Reply:
My understanding is that a consumer of @turf/turf would now see `type: "module"` and infer that turf.min.js is ESM and not UMD because of the .js extension. Is that incorrect?

Reply:
I believe that's incorrect. Someone using @turf/turf from a node project shouldn't have any idea turf.min.js even exists. Node will choose the file to import (dist/esm/index.js or dist/cjs/index.js) based on the import mechanism their project employs. Us fleshing out the exports field in package.json allows that to work more predictably.

Reply:
ok sweet. thanks for the education here

Reply:
Hey @mfedderly remember a few days ago when you thanked me for educating you? Yeah, you are entitled to a refund on that. I was wrong. Half wrong, anyway.

Using turf.min.js should still be unimpacted by whether the monorepo package is type: module or commonjs

What doesn't work perfectly is using ```type: module``` and dist/cjs/index.js (.js extension), even if the exports section in package.json says _that's_ the file to use if you're using require. From the looks of arethetypeswrong, it also needs to have a .cjs extension, or we go back to dropping a ```type: commonjs``` package.json file into dist/cjs/ during the build.